pandas/core/frame.py

```diff
@@ -1071,7 +1077,9 @@ def iterrows(self) -> Iterable[Tuple[Label, Series]]:
             s = klass(v, index=columns, name=k)
             yield k, s
 
-    def itertuples(self, index: bool = True, name: Optional[str] = "Pandas"):
+    def itertuples(
+        self, index: bool = True, name: Optional[str] = "Pandas"
+    ) -> Iterable[Tuple]:
```

Comment:
`Iterable[NamedTuple]`?

Reply:
I think you can put `Iterable[Tuple[Any, ...]]` so that we don't have a generic type without type parameters. Since the tuple/namedtuple is data dependent, we won't be able to refine further.